test/helpers/application_helper_test.rb

```diff
@@ -700,11 +700,23 @@ def render
       assert html.html_safe?
     end
 
-    it "renders succeeded deploys" do
+    it "renders current, succeeded deploys" do
       html = deployed_or_running_list(stage_list, "staging")
```

Comment:
Might be nice to use `assert_sql_queries` here to lock in the number of queries in case someone comes along and changes this later.

Reply:
:+1: